maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/AbstractSurefireMojo.java

```diff
@@ -732,7 +732,8 @@
      * The child elements of this element must be &lt;dependency&gt; elements, and the
      * contents of each of these elements must be a string which follows the format:
      *
-     * <i>groupId:artifactId</i>. For example: <i>org.acme:project-a</i>.
+     * <i>groupId:artifactId[:version[:type[:classifier]]]</i>. For example: <i>org.acme:project-a</i>,
+     * <i>org.acme:project-a::test-jar</i>, <i>org.acme:project-a:::tests-jdk15</i>
```

Comment:
Did the previous pattern `groupId:artifactId` support `*` wildcard. Does your algorithm support `*`? I know that other plugins usually support it with `*`. If both, empty character between :: and `*`, would work, even better.